nsv13/code/modules/munitions/ammunition/broadside_ammo.dm

```diff
@@ -125,6 +155,9 @@
 				plasma = TRUE
 				to_chat(user, "<span class='notice'>You add [A] to the table.</span>")
 			else if(istype(A, /obj/item/powder_bag))
```

Comment:
You can load plasma and gunpowder bags interchangibly with standard loads, while at least one being loaded will create a plasma shell.